packages/-build-infra/src/features.js

```diff
@@ -1,11 +1,10 @@
 'use strict';
 
+const requireEsm = require('esm')(module);
 function getFeatures() {
-  const features = {
-    SAMPLE_FEATURE_FLAG: null,
-    RECORD_DATA_ERRORS: null,
-    RECORD_DATA_STATE: null,
-  };
+  const { DEFAULT_FEATURES: features } = requireEsm(
```

Comment:
NOTE: It was unclear reading esm's documentation whether this would affect later `require` uses.

Reply:
This needs tweaking right? Since the module now exports default, I would think you'd need:

```suggestion
  const { default: features } = requireEsm(
```

Right?